api/src/routes/users/@me/guilds.ts

```
@@ -21,7 +21,7 @@ router.get("/", async (req: Request, res: Response) => {
 // user send to leave a certain guild
 router.delete("/:id", async (req: Request, res: Response) => {
 	const guild_id = req.params.id;
-	if (process.env.INSTANCE_GUILD_ID === guild_id) throw new HTTPError("You can't leave this guild", 400);
```

This is important as 400 on leave guild returns a silent error on some clients. They never expect their own leave request be ill-formed.